students/alex_whitty/lesson2/grid_func.py

```diff
@@ -0,0 +1,62 @@
+
+def plus():
+    plus = '+'
+    return plus
```

Comment:
This function doesn't really gain you anything as its just outputing `+`.  So on line 18 you are actually typing more by doing `plus()` than `+`